src/cmds/scripts/pbs_habitat.in

```diff
@@ -39,6 +39,7 @@ if [ $# -eq 1 ] && [ $1 = "--version" ]; then
    exit 0
 fi
 
+PBS_VERSION="@PBS_VERSION@"
```

Comment:
Double quotes allow for variable expansion. While there shouldn't be a '$' in the version string, it could become a problem.